omod/src/main/webapp/mapLocations.jsp

```diff
@@ -32,6 +34,14 @@
 			<tr>
 				<td />
 				<td><input value="Map Locations" name="submit" type="submit"></td>
+                                </c:if>
+                                    <c:if test="${empty orgunits}">
+                                    <div id="openmrs_msg"><spring:message code="dhisreport.orgUnitCodeDoesNotExist" /> </div>
+                                </td>
+                                 <tr>
+ 				<td />
+                                <td><input value="Map Locations" name="submit" type="submit" disabled></td>
```

Comment:
Is the above line also supposed to be under the if orgunits is empty test?

Reply:
@Choxmi can you point me to the tr that close each of these?
https://github.com/Choxmi/openmrs-module-dhisreport/blob/f762e0e33ca74279199dcc69d684d7f4fc8baf01/omod/src/main/webapp/mapLocations.jsp#L19

https://github.com/Choxmi/openmrs-module-dhisreport/blob/f762e0e33ca74279199dcc69d684d7f4fc8baf01/omod/src/main/webapp/mapLocations.jsp#L34

https://github.com/Choxmi/openmrs-module-dhisreport/blob/f762e0e33ca74279199dcc69d684d7f4fc8baf01/omod/src/main/webapp/mapLocations.jsp#L41

https://github.com/Choxmi/openmrs-module-dhisreport/blob/f762e0e33ca74279199dcc69d684d7f4fc8baf01/omod/src/main/webapp/mapLocations.jsp#L50

Reply:
Yes @dkayiwa I am